core/src/test/java/io/confluent/rest/metrics/MetricsResourceMethodApplicationListenerIntegrationTest.java

```diff
@@ -283,6 +286,61 @@ public void test429Metrics() throws InterruptedException {
     assertEquals(1, windowCheckpoint429); ///A single windowed metric for the two 4xx errors
   }
 
+  @DisplayName("WithGlobalStatsRequestTagsEnabled")
+  @Test
+  public void test429Metrics_WithGlobalStatsRequestTagsEnabled() throws InterruptedException {
```

Comment:
Can you add a comment to the effect? 
`This tests validates that with METRICS_GLOBAL_STATS_REQUEST_TAGS_ENABLE_CONFIG enabled true, the request-tags "value1" "value2" are on the global metrics for 429`
It wasn't obvious to me.